client/my-sites/importer/newsletter/content-upload/author-mapping-pane.jsx

```diff
@@ -124,6 +133,68 @@ class AuthorMappingPane extends PureComponent {
 		}
 	};
 
+	getUnsupportedFilesMessage() {
+		const { translate } = this.props;
+		const unsupportedFiles = this.props.importerStatus?.customData?.unsupportedFileTypes;
+
+		if ( ! unsupportedFiles ) {
+			return null;
+		}
+
+		const fileTypes = Object.entries( unsupportedFiles );
+		if ( fileTypes.length === 0 ) {
+			return null;
+		}
+
+		const formattedTypes = fileTypes.map( ( [ type, count ] ) => {
+			/* translators: %(count)d is the number of files, %(type)s is the file extension (e.g. "avif", "svg") */
+			return translate( '%(count)d .%(type)s image', {
+				args: {
+					count: formatNumber( count ),
```

Comment:
@chriskmnds 👋 If you have a chance, can you check me on this number formatting? I'm mapping them here and then passing them to another translation below. Let me know if there's a better way!

Reply:
@chriskmnds Thank you! Fixed in https://github.com/Automattic/wp-calypso/pull/104222/commits/87e795e1f5946f4c72f51c4d0dd39644aa0af021